### ISSUE TITLE
User messages font colour not showing up in Chat window

### ISSUE THREAD

**Issue by derimmer** [vasl-developers/vasl#1252]
In 6.6.4, players could set a different font colour for their text messages versus their opponent's messages. This was very useful for PBEM games apparently. 

I seem to have broken this in 6.6.5.

**Comment by derimmer** [comment]
@mtl1971 Morris, give your good work on other chat issues, it would be much appreciated if you could have a look at this.

**Comment by mtl1971** [comment]
Ok - this should do it:
https://github.com/vasl-developers/vasl/pull/1264

 - user fonts + colors selected now used for user names and free text messages
 
![image](https://user-images.githubusercontent.com/67708834/205724061-c9f7ab6d-fbb9-4290-afc4-fe8548aaaa1c.png)

with the other parts of the message using the fonts + colors selected for them.

I've also tidied up the code a bit, and marked a few methods as deprecated - I can't see anything using them, but I'm not sure if these are service methods provided to extensions - could you have a quick look to see if you recognize them?

**Comment by derimmer** [comment]
Testing this now . . . user message font colours for sender and receiver now align with their preferences. Working for user names and free text messages. Thanks!

In terms of your code cleanup, first of all thanks Morris for doing a ton of work on this. Much appreciated. In terms of the deprecated methods, the first two I saw are indeed linked to an actively used extension to add special SASL DR and dr. So, they need to stay. I noticed a couple of others which were called something like ParseUserMsg . . . these can be deleted as they are no longer used. I would suggest that we mark them for deletion in November 2023. That will give us time to get all your changes out in 6.6.6 and be sure that there are no other Chatter issues that we need to fix. 

I am going to push the changes to develop.
